andersoncd/solver.py

```diff
@@ -106,6 +104,7 @@ def solver_path(X, y, penalty, eps=1e-3, n_alphas=100, alphas=None,
     # X_dense, X_data, X_indices, X_indptr = _sparse_and_dense(X)
 
     if alphas is None:
+        # TODO pass datafit.gradient at 0
```

Comment:
could this be  done in this PR ?

Reply:
Is it clear that all penalties do have an alpha_max ?